CreateCardForIssueFromLabel/ReadMe.md

```diff
@@ -0,0 +1,56 @@
+# CreateCardForIssueFromLabel
+
+Create card for issue in a project column determined by the issue label. 
```

Comment:
`Create **a** card for **an** issue in a project column determined by the issue label.`

If it works well on Windows, I would recommend you use Grammarly at OS level. It catches these missing articles most of the time.

Reply:
Sometimes it's over-adding those compared to native speakers :/